sources/packages/api/src/utilities/sqlLoader.ts

```diff
@@ -7,8 +7,7 @@ const sqlDirPath = "src-sql";
  * @description Get SQL dir path
  * @returns The SQL files container dir from resources folder
  */
-export const getSQLDirPath = () =>
```

Comment:
Sure that it works without this in windows also? I remember facing an issue while trying to remove this. I may be wrong but vaguely remember.